packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.vue

```diff
@@ -146,6 +154,14 @@ $input-size: var(--onyx-spacing-md);
       border-radius: 100%;
     }
   }
+
+  &__loading {
+    color: var(--onyx-color-text-icons-primary-intense);
+    width: 1rem;
+    height: 1rem;
+    margin: var(--onyx-spacing-sm);
+    align-self: center;
```

Comment:
Why is this needed?

Reply:
This is how it looks like without it

![image](https://github.com/SchwarzIT/onyx/assets/60856270/75d7bce4-24b5-4761-a009-b9807b36c234)